### DISCUSSION TITLE
How do I get the current Pointer position (on the desktop) without responding to an event

### DISCUSSION THREAD

**Discussion by andy-westacott** [AvaloniaUI/Avalonia#15769] [Q&A]
I would like to know where the mouse is at a given point of time and it maybe outside of the Window.  
How can I capture the current mouse position in this case?
Thanks

**Comment by stevemonaco** [comment]
Avalonia doesn't expose raw input APIs, partially because the capabilities won't be available on all targets, especially mobile and WASM. This would require raw input querying as app windows don't receive events for input events that happen outside of the app window with a few exceptions.

You can try something like [SharpHook](https://github.com/TolikPylypchuk/SharpHook).
